packages/components/pie-assistive-text/src/index.ts

```diff
@@ -13,8 +19,36 @@ const componentSelector = 'pie-assistive-text';
  * @tagname pie-assistive-text
  */
 export class PieAssistiveText extends LitElement implements AssistiveTextProps {
+    @property()
+    @validPropertyValues(componentSelector, variants, 'default')
+    public variant?: AssistiveTextProps['variant'] = 'default';
+
+    /**
+ * Renders the assistive-text icon content.
+ * @private
+ */
+    private renderContent (): TemplateResult {
+        const { variant } = this;
+        return html`
+            ${variant === 'success' ? html`<icon-check-circle size="s" />` : nothing}
+            ${variant === 'error' ? html`<icon-alert-circle size="s" />` : nothing}`;
+    }
+
     render () {
-        return html`<h1 data-test-id="pie-assistive-text">Hello world!</h1>`;
+        const {
+            variant,
+        } = this;
+
+        return html`
+        <p 
+            class="c-assistiveText"
+            data-test-id="pie-assistive-text"
+            variant=${variant || 'default'}>
```

Comment:
would `ifDefined(variant)` work here? I'm not super keen on repeating the default string in the template (or you could use a default values object like in `pie-input`?)

Realistically the chances of it actually being undefined are very low, even in a JS consuming application

Reply:
just tested and `ifDefined` does indeed work here! I agree it's nicer than duplicating the default string, changing now 😄